tests/write_new_mask.phpt

```diff
@@ -1,7 +1,7 @@
 --TEST--
 Check writing to files in new dir with dir mask
 --INI--
-shadow.mkdir_mask = 0755
```

Comment:
everything is okay, but that one we should leave to be failed because it shows real problem that ini changes don't work after shadow initialization.